src/three-components/SmoothControls.ts

```diff
@@ -400,6 +401,15 @@ export class SmoothControls extends EventDispatcher {
     }
   }
 
+  private get[$canInteract](): boolean {
+    if (this[$options].interactionPolicy === 'always-allow') {
+      return true;
+    }
+
+    const rootNode = this.element.getRootNode() as Document | ShadowRoot;
+    return rootNode.activeElement === this.element;
```

Comment:
Maybe should ensure that the interaction policy is 'allow-when-focused' (had to check SmoothControls to see if these were the only two valid tokens for interaction policy)

Reply:
👍 will fix